docs/contributing/using-instrumenter-api.md

```diff
@@ -116,6 +116,33 @@ The `builder()` method accepts three arguments:
 An `Instrumenter` can be built from several smaller components. The following subsections describe
 all interfaces that can be used to customize an `Instrumenter`.
 
+### Set the instrumentation version and OpenTelemetry schema URL
+
+Make sure you always provide the instrumentation library version to the `Instrumenter`. You can do
```

Comment:
@theletterf I've added a few more paragraphs, it'd be great if you could proofread them too 🙏